packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-list-card.tsx

```diff
@@ -182,57 +186,204 @@ export const PropertyTypeRow = ({
   propertyIndex: number;
   onRemove: () => void;
 }) => {
-  const { watch } = useFormContext<EntityTypeEditorForm>();
+  const { control, watch } = useFormContext<EntityTypeEditorForm>();
+  const { update } = useFieldArray({
+    control,
+    name: "properties",
+  });
+
   const propertyTypes = usePropertyTypes();
-  const propertyId = mustBeVersionedUri(
-    watch(`properties.${propertyIndex}.$id`),
-  );
+  const [anchorEl, setAnchorEl] = useState<HTMLDivElement | null>(null);
+  const [multipleValuesMenuOpen, setMultipleValuesMenuOpen] = useState(false);
+
+  const draftProperty = watch(`properties.${propertyIndex}`);
+
+  const { $id, array, minValue, maxValue, required } = draftProperty;
+
+  const propertyId: VersionedUri = mustBeVersionedUri($id);
   const property = propertyTypes ? propertyTypes[propertyId] : null;
 
-  if (!property) {
-    throw new Error("Missing property type");
-  }
+  // if (!property) {
+  //   throw new Error("Missing property type");
+  // }
```

Comment:
What's this for? Would be a lot cleaner diff to make this 

```
if (!property) {
   return null;
}
```

Instead of the ternary below 

But unsure how rendering a `PropertyTypeRow` for a missing property isn't an error

Reply:
This was throwing when I reloaded the page of an already created entity type, because `propertyTypes` needed to load first. I think c498413131cbac6f3c0329b301a6badae7062674 addresses this nicely but let me know what you think.